plugins/vmware_esxi.yaml

```diff
@@ -0,0 +1,30 @@
+# Plugin Info
+version: 0.0.1
+title: VMware ESXi
+description: Log parser for VMware ESXi
+parameters:
+  listen_address:
+    label: Listen Address
+    description: A syslog address of the form `<ip>:<port>`
+    type: string
+    default: ":5140"
+
+# Set Defaults
+{{$listen_address := default ":5140" .listen_address}}
+
+# Pipeline Template
+pipeline:
+  - id: esxi_input
+    type: tcp_input
+    listen_address: {{ $listen_address }}
+    labels:
+      log_type: vmware_esxi
+    output: esxi_parser
+
+  - id: esxi_parser
+    type: regex_parser
+    regex: '^(?P<timestamp>[a-zA-z]+ \d{2} \d{2}:\d{2}:\d{2}) (?P<hostname>[^ ]+) (?P<service>[^ ]+) (?P<message>.*)'
```

Comment:
Is this not syslog?

Reply:
This is the same problem we ran into with vmware vcenter and cisco meraki.